CONTRIBUTING.md

```diff
@@ -85,6 +85,15 @@ NuGet members may contribute directly to the main remote.
 
     > Note: You have to to run .\configure.ps1 and .\build.ps1 at least once in order for your build to succeed.
 
+1. Run dotnet code formatters and correct any errors.
```

Comment:
Not sure this needs to be an extra step, before writing the tests. 

If anything it'd be after. 

I'd recommending adding the check to the build.ps1 script so people locally can repro what we do on the CI. 

This is *not* the only issue people might run into.

Reply:
You mean before running the tests?
I would argue you don't want to run tests, then have to modify code due to formatting. Why wait?

Reply:
You'll have to iteration as you run tests to fix issues. Ideally the code should be formatted as a final step when *all* code changes are completed.

Reply:
We can consider adding this as an opt-in feature in the build script if we don't want to break local builds due to formatting every time. By default we run format check and community members can invoke something like `build.ps1 -skipFormatCheck` to skip format check during inner loop.

Reply:
I'm not sure I want to run all my tests, only to find out I need to change a bunch of code files for formatting purposes and have to rerun them.
But I don't care where we put it as long as it's at least mentioned. Devs will do their process however they want...

Reply:
Moved to last step